### DISCUSSION TITLE
Gradio tab to start a model does not load

### DISCUSSION THREAD

**Discussion by vitalijmirnyj** [TheLastBen/fast-stable-diffusion#1760] [Q&A]
When starting or testing model, the link which should open gradio in new tab does not load. I tried deleting everything from my drive, does not help. It worked today in the morning. Thank you for your work.

**Comment by martindubo** [comment]
The same thing happens to me, I get "504 Gateway Time-out" is there a way to fix it?

**Comment by THX1139b** [comment]
Same here. I also had trouble getting anything to work the last time I tried a few days ago. First img2img stopped working altogether and then text2img began acting up as well. And now it won’t even load.

My earlier problems may of course be unrelated but it feels like things have gotten gradually less stable. I too have deleted the old DS foler and run an updated notebook, installed ControlNet again etc.

Hoping this isn’t the end since I love this and need it to feel like a complete person. :)

**Comment by EliotVarda** [comment]
i was having the same problem so i tried it on firefox and it worked
or maybe it was a coincidence and the bug is fixed by now

**Reply by meunumerotim333** [reply]
Same problem on Firefox, just tried

![image](https://user-images.githubusercontent.com/121369033/224795596-0d6865ba-dee0-4248-a4bd-dcab9ff10766.png)

**Comment by meunumerotim333** [comment]
For me is also a lot buggy. I'm trying for 20 minutes load a prompt on PNG info and I'm just getting **error** message. I've also tried use Local Tunnel and the problem is the same.

**Comment by vitalijmirnyj** [comment]
It is working now. Thank you

**Reply by THX1139b** [reply]
Not for me it isn't. Tried inpainting and it won't load the image. Also there have been no updates for two days so it seems illogical that anything has changed yet.

**Comment by orlandoai23** [comment]
I'm having very similar issues.  Yesterday it was giving me a 504, but then eventually worked. 

Today, it takes several minutes for Gradio to even load. Once it loads, I click "Generate". It gets to a little over 20% complete (can see the images being created), then all the images disappear (WebUi image preview goes blank).

However, when I look at the code, it's still running and doing my batches with the 30 sampling steps (see image). So, it's obviously still running in the background, but not loading the images in the UI. 

Any ideas?

![image](https://user-images.githubusercontent.com/127401105/225032120-eb3b186a-41de-4976-b440-06d89d14b2fc.png)

**Reply by orlandoai23** [reply]
They ended up eventually rendering, but now I'm downloading as a zip and the download process on a 37MB file is taking FOREVER. Clearly there are issues.

**Reply by THX1139b** [reply]
I'm trying it again with today's release. Deleting old files first as per usual. I get it to load a picture once in a while but mostly it times out. But some of them do render eventually.

Since the status bar tends to always stop working after a few iterations, even when there are no error messages, it is difficult to know if an image will eventually appear. Most of the time now it never actually renders anything (img2img) but today an image did appear a couple of times. But after squeezing out six images in total between the errors, it seems to be over and there are no more.

AssertionError it says when everything stops.

**Reply by orlandoai23** [reply]
It now seems to be working and the UI is back to working again. Just blips in the interwebs I guess. :)

**Comment by meunumerotim333** [comment]
I just would like the UI runs without bugs as before.. last year the running was clear. Hope Ben have some time to dedicate and fix those bugs

**Reply by TheLastBen** [reply]
those errors are out of my control, they are related to gradio servers or colab internal structure, can't change that

**Reply by THX1139b** [reply]
Be that as it may, unfortunately it means that this way of running Automatic1111 is currently not usable. This is a real shame as it has been a wonderful way of using SD for a Mac user like me.

**Reply by TheLastBen** [reply]
try RunPod when colab is unstable

**Reply by THX1139b** [reply]
Yes, I need to figure out how to use that. Something new to learn. Thanks!

**Reply by TheLastBen** [reply]
let me know if you need guidance
